mesontest.py

```diff
@@ -101,9 +101,10 @@ def get_log(self):
         if self.cmd is None:
             res += 'NONE\n'
         else:
-            res += "\n%s %s\n" %(' '.join(
-                ["%s='%s'" % (k, v) for k, v in self.env.items()]),
-                ' ' .join(self.cmd))
+            res += "%s%s\n" % (
+                    ''.join(["%s='%s' " % (k, v) for k, v in self.env.items()]),
+                    ' ' .join(self.cmd)
+                )
```

Comment:
__[flake8]__
* [E121] continuation line under-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->